src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -104,12 +135,13 @@ class Lambda {
     dotcomSurrogateKey
   }
 
-  private def sendFastlyPurgeRequestAndAmpPingRequest(contentId: String, purgeType: PurgeType, serviceId: String, surrogateKey: String, fastlyApiKey: String): Boolean = {
-    if (sendFastlyPurgeRequest(contentId, purgeType, serviceId, surrogateKey, fastlyApiKey))
-      AmpFlusher.sendAmpDeleteRequest(contentId)
-    else
-      false
-  }
+  // TODO: This appears to be unused - delete it?
+  //  private def sendFastlyPurgeRequestAndAmpPingRequest(contentId: String, purgeType: PurgeType, serviceId: String, surrogateKey: String, fastlyApiKey: String): Boolean = {
```

Comment:
I (or IntelliJ anyway) couldn't find anything triggering this, but I know @shtukas was working on AMP stuff recently so he may have info.

Reply:
I believe this was used in the main branch; I've moved it's special AMP call to the a post decache action next to the SNS events publishing.